### PR TITLE
Added some modern tech jobs

### PR DESCRIPTION
**Pull request by flomerboy**
To make the data a little more usable, added some modern tech jobs, still under 1000
Please disregard if I'm missing something that makes this a bad idea. I read the readme.

### CODE REVIEW

File: data/humans/occupations.json [hunk 1]
```diff
@@ -942,6 +946,7 @@
 		"upholsterer",
 		"urban planner",
 		"usher",
+    "ux designer",
```
Comment (hugovk) [inline data/humans/occupations.json:949]: Shall we put this uppercase for clarity? People can convert to lower case later if they like. 
```suggestion
    "UX designer",
```

**Comment by flomerboy** [comment]
Seems logical to me!

On October 30, 2021, Zach H ***@***.***> wrote:
> @hugovk commented on this pull request.
>

> In data/humans/occupations.json
> <https://github.com/dariusk/corpora/pull/386#discussion_r739637179>:
>

> @@ -942,6 +946,7 @@ "upholsterer", "urban planner", "usher", + "ux
designer",
> Shall we put this uppercase for clarity? People can convert to lower
> case later if they like.
>  ⬇️ Suggested change 

- "ux designer", + "UX designer",

**Comment by hugovk** [comment]
Thank you!

**Review by flomerboy — COMMENTED** [review]
lgtm

**Pull request merged**